src/main/java/ch/njol/skript/util/SkriptColor.java

```diff
@@ -219,7 +221,18 @@ public static SkriptColor fromWoolData(short data) {
 		}
 		return null;
 	}
-	
+
+	/**
+	 * Replace chat color character '§' with '&'
+	 * This is an alternative method to {@link ChatColor#stripColor(String)}
+	 * But does not strip the color code.
+	 * @param s string to replace chat color character of.
+	 * @return String with replaced chat color character
+	 */
+	public static String replaceColorChar(String s) {
+		return COLOR_CHAR_PATTERN.matcher(s).replaceAll("&");
```

Comment:
```suggestion
		return s.replace('\u00A7', '&');
```
I think it's better to use this form to avoid any possible encoding issues